Add cocotb.tops to expose all top-level design units (#5371)
cocotb/cocotb: docs/source/newsfragments/5371.feature.rst

> Added ``cocotb.tops`` to expose a dictionary of all top-level design units for the Icarus simulator.

Reviewer: ```suggestion
Added :data:`cocotb.tops` to expose a dictionary containing all top-level design units.
```
